tests/manage/test_create_storageclass_with_wrong_provisioner.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+import logging
+
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.framework.testlib import ManageTest, tier3
+from tests.fixtures import (
+    create_ceph_block_pool,
+    create_rbd_secret, create_cephfs_secret,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier3
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_ceph_block_pool.__name__,
+)
+class TestCreateStorageClassWithWrongProvisioner(ManageTest):
+    """
+    Create Storage Class with wrong provisioner
+    """
+    @pytest.mark.parametrize(
+        argnames="interface",
+        argvalues=[
+            pytest.param(
+                *["RBD"], marks=pytest.mark.polarion_id("OCS-620")
+            ),
+            pytest.param(
+                *["CEPHFS"], marks=pytest.mark.polarion_id("OCS-621")
+            )
+        ]
+    )
+    def test_create_storage_class_with_wrong_provisioner(self, interface):
+        """
+         Test function which creates Storage Class with
+         wrong provisioner and verifies PVC status
+        """
+        log.info(f"Creating a {interface} storage class")
+        self.sc_data = templating.load_yaml_to_dict(
+            getattr(constants, f"CSI_{interface}_STORAGECLASS_YAML")
+        )
+        self.sc_data['metadata']['name'] = (
+            helpers.create_unique_resource_name(
+                'test', f'csi-{interface.lower()}'
+            )
+        )
+        self.sc_data['provisioner'] = "openshift.org/aws-efs"
+        SC_OBJ = OCS(**self.sc_data)
+        assert SC_OBJ.create()
+        log.info(
+            f"{interface}Storage class: {SC_OBJ.name} created successfully"
+        )
+        log.info(self.sc_data)
+
+        # Create PVC
+        PVC_OBJ = helpers.create_pvc(sc_name=SC_OBJ.name, wait=False)
+
+        # Check PVC status
+        pvc_output = PVC_OBJ.get()
```

Comment:
create_pvc will return the updated pvc data as OCS.create() includes reload()  - ocs_ci/ocs/resources/ocs.py:92